tests/rgen/Microsoft.Macios.Generator.Tests/Classes/Data/ExpectedAVAudioPcmBufferNoDefaultCtr.cs

```diff
@@ -0,0 +1,28 @@
+// <auto-generated />
+
+#nullable enable
+
+using Foundation;
+using ObjCBindings;
+using ObjCRuntime;
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+using System.Threading.Tasks;
+
+namespace TestNamespace;
+
+[Register ("AVAudioPCMBuffer", true)]
+public partial class AVAudioPcmBuffer
+{
+	[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
+	static readonly NativeHandle class_ptr = Class.GetHandle ("AVAudioPCMBuffer");
+
+	public override NativeHandle ClassHandle => class_ptr;
+
+	// TODO: add binding code here
```

Comment:
The protected (NSObjectFlag) and (NativeHandle) ctors should still be generated iirc.